plugins/train/_config.py

```diff
@@ -41,21 +41,13 @@ def set_globals(self):
         section = "global"
         self.add_section(title=section,
                          info="Options that apply to all models" + ADDITIONAL_INFO)
-        self.add_item(section=section, title="mask_type", datatype=str, default="none",
-                      choices=get_available_masks(), gui_radio=True,
-                      info="The mask to be used for training:"
-                           "\n\t none: Doesn't use any mask."
-                           "\n\t components: An improved face hull mask using a facehull of 8 "
-                           "facial parts"
-                           "\n\t dfl_full: An improved face hull mask using a facehull of 3 "
-                           "facial parts"
-                           "\n\t extended: Based on components mask. Extends the eyebrow points "
-                           "to further up the forehead. May perform badly on difficult angles."
-                           "\n\t facehull: Face cutout based on landmarks")
         self.add_item(
             section=section, title="icnr_init", datatype=bool, default=False,
-            info="Use ICNR Kernel Initializer for upscaling.\nThis can help reduce the "
-                 "'checkerboard effect' when upscaling the image.")
+            info="\nUse ICNR to tile the default initializer in a repeating pattern. \n"
+                 "This strategy is designed for  sub-pixel / pixel shuffler upscaling \n"
+                 "and should only be used on upscaling layers. This can help reduce the \n"
```

Comment:
I'm not sure you need to say that it should only be used on upscaling layers here. This is an end user option, so it will either be meaningless or confusing to most. A better place might be in the docstring for the function

Reply:
K, I'll move that around to docstring